src/core.c/CompUnit/PrecompilationRepository.pm6

```diff
@@ -157,9 +157,9 @@ Need to re-check dependencies.")
             $resolve := True;
         }
 
-        if $unit-id ne $first-repo-id {
+        if $REPO-id ne $first-repo-id {
```

Comment:
I'm confused, because if we look back at the code before the rechecking started occurring we see it using `$unit-id` (`$repo-id` in the code linked)

https://github.com/rakudo/rakudo/blob/f5ce80e1a6d60a6a51a898da85f3100bb088dfb4/src/core.c/CompUnit/PrecompilationRepository.pm6#L112-L120

Reply:
Oops nevermind, I read `self!load-repo-id` as `self!load-file`